code/modules/mob/living/carbon/human/species.dm

```diff
@@ -142,11 +226,24 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 
 	return randname
 
-//Called when cloning, copies some vars that should be kept
+/**
+  * Called when cloning, copies some vars that should be kept
+  *
```

Comment:
```suggestion
/**
```
Cloning is kill

Reply:
Yeah but this is still used by some species, like slime dudes

Reply:
Yes leave that bit I'm just saying to remove the reference to cloning

Reply:
cloning is what i'd best describe what this proc does even if cloning is kill

Reply:
"Called when cloning" describes when the proc is called and not what it does, so at least remove that bit, though "copies some vars that should be kept" can be left in

Reply:
i guess that makes sense, tivi